rcnn/rcnn.cpp

```diff
@@ -510,18 +524,29 @@ int main(int argc, char** argv) {
         auto end = std::chrono::system_clock::now();
         std::cout << std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count() << "ms" << std::endl;
 
-        float h_ratio = static_cast<float>(INPUT_H) / IMAGE_HEIGHT;
-        float w_ratio = static_cast<float>(INPUT_W) / IMAGE_WIDTH;
+        float h_ratio = static_cast<float>(h_ori) / (INPUT_H - Y_PAD * 2);  // ratio of original image size to model input size
```

Comment:
is't ok if batch!=1, maybe h_ori will be different with each other batch

Reply:
Yes. If batch!=1, each h_ori may be different. So, I will fix this var.

Reply:
Yes, I will modify it according to your above thinking of padding.